From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 11 May 2016 01:54:14 +0200
Subject: add_missing_header_guards

---
 src/groupwareuidelegate.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/groupwareuidelegate.h b/src/groupwareuidelegate.h
index 5f44247..293a945 100644
--- a/src/groupwareuidelegate.h
+++ b/src/groupwareuidelegate.h
@@ -16,6 +16,9 @@
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 */
 
+#ifndef GROUPWAREUIDELEGATE_H
+#define GROUPWAREUIDELEGATE_H
+
 #include "incidenceeditor_export.h"
 
 #include <Akonadi/Calendar/ITIPHandler>
@@ -34,3 +37,4 @@ public:
 };
 
 }
+#endif // GROUPWAREUIDELEGATE_H
