From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 11 May 2016 01:54:14 +0200
Subject: add_missing_header_guards

---
 src/groupwareuidelegate.h | 4 ++++
 1 file changed, 4 insertions(+)

Index: b/src/groupwareuidelegate.h
===================================================================
--- a/src/groupwareuidelegate.h
+++ b/src/groupwareuidelegate.h
@@ -16,6 +16,9 @@
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 */
 
+#ifndef GROUPWAREUIDELEGATE_H
+#define GROUPWAREUIDELEGATE_H
+
 #include "incidenceeditor_export.h"
 
 #include <Akonadi/Calendar/ITIPHandler>
@@ -31,3 +34,4 @@
     void createCalendar() override;
 };
 }
+#endif // GROUPWAREUIDELEGATE_H
